### DISCUSSION TITLE
Removing IE 11 support in v7

### DISCUSSION THREAD

**Discussion by TrueWill** [react-hook-form/react-hook-form#4396] [General] [answered]
The next release is removing support for Internet Explorer 11. While no one (I hope) _likes_ IE, this is a dealbreaker for many corporate developers (myself included). Is there any possibility IE11 could still be supported through polyfills and/or transpilation? Thank you!

**Comment by bluebill1049** [comment, marked as answer]
you will have to include your own corejs or stay with v6. our relationship with IE 11 ends in v7.

**Comment by JustinLien** [comment]
Following.

@bluebill1049 does this mean as long as you include corejs, the v7 will work fine in IE11?

**Reply by bluebill1049** [reply]
unfortunately note, you will have to use v6.

**Reply by Joshgrant-dev** [reply]
Hey following on this discussion too,

@bluebill1049 

Do you have an implementation guide of how to use v6 with IE 11? Is there any extra configuration I have to do other than import 'corejs' and downgrade to v6? 

I will say it's a shame I won't get the fully maintained react-hook-form, because the library is overall fantastic! However also in the position where we need to support IE.

**Reply by JustinLien** [reply]
@Joshgrant-dev

Not sure the example was removed, but for IE11 support in v6, you'll need to import ie11 version.

```
import { useForm } from 'react-hook-form/dist/index.ie11'; // V6
import { useForm } from 'react-hook-form/dist/react-hook-form.ie11'; // V5'
// Resolvers
import { yupResolver } from '@hookform/resolvers/dist/ie11/yup';
```

https://react-hook-form.com/faqs#BrowserSupport
Additionally, you'll need to add this as well.

`npm i @babel/runtime-corejs3`

And any of those methods listed.

.values, .entries, .flat, .find, .includes, and .startsWith

**Reply by bluebill1049** [reply]
Thanks @JustinLien

**Reply by Joshgrant-dev** [reply]
Brilliant! Thank you very much for the excellent support and fast response with this! Fingers crossed it goes smoothly to downgrade to v6!

**Reply by betweenparentheses** [reply]
we are using v6 right now without importing a specific ie11 version, but it still passes our crossbrowser testing with ie11. any insight into that?

**Reply by bluebill1049** [reply]
> we are using v6 right now without importing a specific ie11 version, but it still passes our crossbrowser testing with ie11. any insight into that?

Huh... Not sure, I think u guys must have all the ployfill already installed.
